backend/app/assets/javascripts/spree/backend/views/payment/payment_row.js.coffee

```diff
@@ -0,0 +1,24 @@
+Spree.Views.Payment.PaymentRow = Backbone.View.extend
+  events:
+    "click .js-edit": "onEdit"
+    "click .js-save": "onSave"
+    "click .js-cancel": "onCancel"
+
+  onEdit: (e) ->
+    e.preventDefault()
+    @$el.addClass("editing")
```

Comment:
Surely is outside the scope of this changes.

But we have similar backbone views that use a conditional value `editing` with a handlebars template.  (ex: [orders/shipping_methods](https://github.com/solidusio/solidus/blob/926ea6880b232b353d8cea1479a2f1bcc41ac279/backend/app/assets/javascripts/spree/backend/views/order/shipping_method.js#L18)) I think that approach is better than show/hide with css.

Maybe we can create a common view for use in this cases.

Reply:
Absolutely. Will see if I can sneak this into this PR